### PR TITLE
dev: ClickHouse 21.8

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

File: src/sentry/conf/server.py [hunk 1]
```diff
@@ -2732,12 +2726,7 @@ def build_cdc_postgres_init_db_volume(settings: Any) -> dict[str, dict[str, str]
     ),
     "clickhouse": lambda settings, options: (
         {
-            "image": "ghcr.io/getsentry/image-mirror-yandex-clickhouse-server:20.3.9.70"
-            if not APPLE_ARM64
-            # altinity provides clickhouse support to other companies
-            # Official support: https://github.com/ClickHouse/ClickHouse/issues/22222
-            # This image is build with this script https://gist.github.com/filimonov/5f9732909ff66d5d0a65b8283382590d
-            else "ghcr.io/getsentry/image-mirror-altinity-clickhouse-server:21.6.1.6734-testing-arm",
+            "image": "ghcr.io/getsentry/image-mirror-yandex-clickhouse-server:21.8.13.1",
```
Comment (MeredithAnya) [inline src/sentry/conf/server.py:2729]: Im assuming you need the altinity stable at the end https://github.com/getsentry/image-mirror/pkgs/container/image-mirror-altinity-clickhouse-server/115571514?tag=21.8.13.1.altinitystable

**Review by MeredithAnya — APPROVED** [review]

**Review by asottile-sentry — APPROVED** [review]
noice

**Comment by lynnagara** [comment]
@asottile-sentry do you have any idea why all these random tests are failing?

**Comment by lynnagara** [comment]
I marked the bad tests XFAIL, though they should probably just be deleted

**Comment by onewland** [comment]
looking forward to seeing this hit 22.8 🚀

**Pull request merged**